src/NHibernate.Test/NHSpecificTest/NH3374/FixtureByCode.cs

```diff
@@ -15,9 +13,11 @@ protected override HbmMapping GetMappings()
 			mapper.Class<Document>(rc =>
 			{
 				rc.Id(x => x.Id, idMapper => idMapper.Generator(Generators.Identity));
-				rc.ManyToOne(x => x.Blob, m =>
+				rc.ManyToOne(x => x.Blob,
+					m =>
 					{
 						m.Cascade(Mapping.ByCode.Cascade.All);
+						m.Column("`Blob`");
```

Comment:
Those keywords are seldom used in the code base. Only the `Template` class uses them, and not for quoting.